ext_emconf.php

```diff
@@ -11,12 +12,12 @@
     'version' => '1.0.1',
     'constraints' => [
         'depends' => [
-            'typo3' => '9.5.0-10.0.99',
-            'frontend' => '9.5.0-10.0.99'
+            'typo3' => '9.5.0-11.5.99',
```

Comment:
please adapt TYPO3 version to composer.json values

Reply:
all done!